editor/script_create_dialog.cpp

```diff
@@ -35,6 +35,16 @@
 #include "os/file_access.h"
 #include "script_language.h"
 
+void ScriptCreateDialog::_notification(int p_what) {
+
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE: {
+			path_button->set_icon(get_icon("Folder", "EditorIcons"));
```

Comment:
Isn't it possible to set the icon when you instantiate the nodes in the constructor?